phase/download_k0s.go

```diff
@@ -24,15 +24,23 @@ func (p *DownloadK0s) Title() string {
 func (p *DownloadK0s) Prepare(config *v1beta1.Cluster) error {
 	p.Config = config
 	p.hosts = p.Config.Spec.Hosts.Filter(func(h *cluster.Host) bool {
-		if h.Metadata.K0sBinaryVersion == p.Config.Spec.K0s.Version {
+		// Nothing to upload
+		if h.UploadBinary {
```

Comment:
Is there a reason `if h.UploadBinary` and `if h.Metadata.K0sBinaryVersion == p.Config.Spec.K0s.Version` were flipped? Not sure if it matters, just curious.

Reply:
Not sure how that happened, I think from copy pasting parts of the code. But I think the order of those if checks doesn't really matter.